### DISCUSSION TITLE
How to extract interesting cell cluster?

### DISCUSSION THREAD

**Discussion by Ping-lin14** [GreenleafLab/ArchR#640] [Questions / Documentation]
Just like the title.
In my data, in addition to the cells I am interested in, there are many immune cells.
I want to ignore immune cells in the downstream analysis and only keep the cells of interest. 
Can ArchR do that?

**Comment by rcorces** [comment]
You can use `subsetArchRProject()` to create a new ArchR project that only contains your cells / clusters of interest.
https://www.archrproject.com/reference/subsetArchRProject.html

**Reply by Ping-lin14** [reply]
Do you have a demo script? I want to extract subsets based on the tags given again.

**Reply by rcorces** [reply]
What do you mean by "based on the tags"? Project subsetting should be clear from the function definition and the manual. Please review those. https://www.archrproject.com/bookdown/manipulating-an-archrproject.html

**Reply by Ping-lin14** [reply]
I mean the label on UMAP.
